website/siteConfig.js

```diff
@@ -39,6 +39,7 @@ const siteConfig = {
     { doc: 'interop/interop_index', label: 'Interop' },
     { doc: 'usecases/usecases_index', label: 'Use Cases' },
     { doc: 'advanced/advanced_index', label: 'Advanced' },
+    { doc: 'resources/resources', label: 'Resources & Ecosystem' },
```

Comment:
I think it may be a good idea to split this up into `Resources` and `Ecosystem` separately. Resources is for people who want books, talks, blogs, etc. Ecosystem is a quick check: if I choose this library, what else works with it right now.